src/standalone/Bucket/Bucket.jl

```diff
@@ -336,6 +337,7 @@ auxiliary_vars(::BucketModel) = (
     :snow_melt,
     :infiltration,
     :top_bc_wvec,
+    :e_per_area,
```

Comment:
Do we also need liquid water volume per area?

In the soil model, we call these "total_energy" and "total_water". Could we use those names here, or else also change the soil aux names to be "e_per_area" and "liq_water_vol_per_area"?

Reply:
The ClimaCoupler method to compute total water doesn't require allocating a new field in the bucket cache, so we don't have the same need to add it here ([this](https://github.com/CliMA/ClimaCoupler.jl/blob/main/experiments/ClimaEarth/components/land/climaland_bucket.jl#L280-L284) is where it's computed in the coupler). But it could be nice to do the same thing for energy and water, and it would make the access from the coupler simpler. I'm okay with either way - do you have a preference?

I'll rename this to `total_energy` to match the soil!

Reply:
Thanks!

I'm in favor of adding the water term here too since it makes the coupler work a bit simpler